pom.xml

```
@@ -1636,7 +1636,7 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xs
       <dependency>
         <groupId>org.rocksdb</groupId>
         <artifactId>rocksdbjni</artifactId>
-        <version>6.25.3</version>
+        <version>7.0.3</version>
```

```suggestion
        <version>7.0.4</version>
```

Reading through the [7.0.4 changelog](https://github.com/facebook/rocksdb/releases/tag/v7.0.4). We probably should bump this to 7.0.4, once the [mvn repository](https://mvnrepository.com/artifact/org.rocksdb/rocksdbjni) is updated.

---

Yeah it has not show up on MVN yet.

---

I am working towards 6.29.5 and 7.0.4 releases at the moment, but it will take a little more time...